lock-email/src/main/res/values/strings.xml

```diff
@@ -36,13 +38,26 @@
     <string name="com_auth0_email_login_access_btn_text">Login</string>
     <string name="com_auth0_email_code_placeholder">Email Code</string>
     <string name="com_auth0_email_no_code_button">Didn\'t receive code</string>
+    <string name="com_auth0_email_no_magic_link_button">Didn\'t receive the link</string>
+    <string name="com_auth0_email_login_message_magic_link">
+    <![CDATA[
+        Please check your email <b>%s</b>.<br/>You&#39;ve received a message from us with the link
+    ]]>
+    </string>
+    <string name="com_auth0_email_login_message_magic_link_in_progress">
+    <![CDATA[
+        Trying to log in with your email <b>%s</b>.
+    ]]>
+    </string>
     <string name="com_auth0_email_login_message">
     <![CDATA[
         Please check your email <b>%s</b>.<br/>You&#39;ve received a message from us with your passcode
     ]]>
     </string>
+    <string name="com_auth0_email_invalid_link_error_title">This link isn\'t valid anymore</string>
```

Comment:
`Couldn't log in with the magic link, it was probably used already or not valid anymore. Please request a new one`